zboxcore/sdk/sdk.go

```diff
@@ -1404,9 +1404,7 @@ func smartContractTxnValueFee(sn transaction.SmartContractTxnData,
 				zap.Any("txn", txn))
 			return
 		}
-		l.Logger.Info("estimate txn fee",
-			zap.Uint64("fee", fee),
-			zap.Any("txn", txn))
+		l.Logger.Info("estimated txn fee: ", fee)
```

Comment:
We'd better log the txn hash at least so that we can know what is this fee used for.

Reply:
fixed in ac0171a